index.html

```diff
@@ -279,5 +279,35 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
+            <!--Works Section contact page-->
+            <section class="page5">
+              <div class="@continer">
+                <div class="bag">
+                <li class="shape1"><img src="image/background/contact-top1.png"></li>
+                <li class="shape2"><img src="image/background/contact-bottom.png"></li>
+                <li class="shape3"><img src="image/background/Capture3.PNG"></li>
+                <div class="h1">
+                  <li class="h2"> <h2>Contact me</h2></li>
+                  <li class="p1"> <p>If you have an application you<br>are interested in developing, a feature that you need built or a project that needs coding.<br> I’d love to help with it!</p></li>
+                  <ul>
+                    <li class="skill1">
+                      <span class="skillـname1">Lorem Ipsum</span>
+                    </li>
+                    </ul>
+                    <ul>
+                      <li class="skill1">
+                        <span class="skillـname1">loremipsum@gmail.com</span>
+                      </li>
+                    </ul>
+                    <ul>
+                      <li class="skill2">
+                        <span class="skillـname2">Write your message here.</span>
+                      </li>
+                    </ul>
+                    <div class="btn4">
+                      <button type="button3" class="btn-get1">Get in touch
+                      </button>
+                    </div>
+                
```

Comment:
- Hey, great job so far, however, we need to restructure your HTML document here to improve your document layout. Some of the opening tags are missing their corresponding closing tags. The document is missing the closing body and html tag.


---

- You should use the html form but you've not used any. let's improve on this.
- The form button by default is a submit button, but you've used `button3` which is semantically not correct. kindly update this.
- The section uses the CSS background property to add the background image but you've used the img to add the background image. kindly update this.
- You've used `ul` multiple times whereas the `ul` is a container for an unordered list and has a `li` for each list item. let's improve on this.

---

*Below is an idea of how this section should look like....*

```html

<section>
  <h2 ></h2>
  <p></p>
  <form action="" method="POST">
    <label for="username">
      <input type="text">
    </label>
  
    <label for="email">
      <input type="email">
    </label>
  
    <label for="message">
      <textarea name="message"></textarea>
    </label>
      <button type="submit"></button>
  </form>
</section>
```

- Please, update it and make sure your form works as expected before submitting it for another review.